meta/src/main/java/org/apache/rocketmq/mqtt/meta/raft/processor/CounterStateProcessor.java

```diff
@@ -62,6 +72,24 @@ public Response onWriteRequest(WriteRequest writeRequest) {
         }
     }
 
+    @Override
+    public SnapshotOperation loadSnapshotOperate() {
```

Comment:
snapshotOperation can be init in constructor， or  init function with @postconstruct

Reply:
However, you should consider that the user must use this framework for snapshot operations. This may be achieved through design such as inheritance